Playbooks 7.0 (#5712)
mattermost/docs: source/playbooks/customize-a-playbook.rst

   - To edit the playbook directly select the actions menu next to the playbook name, then select **Edit**.
   - To access the playbook dashboard, select the hyperlinked playbook name.

4. Select the **Outline** tab. 

> 5. Configure the playbook the same way a playbook is created or edited.

Reviewer: > "... or edited."

This is slightly confusing since this makes me assume that there's another section that explains how a playbook is edited, whereas this is the edit playbook section of the docs. Not sure if I'm interpreting this incorrectly though.

Author: Great catch. I'll revise.

> 5. Configure the playbook the same way a playbook is created or edited.

Author: ```suggestion
5. Configure the playbook inline, the same way you created it.
```
